include/rosdiscover-clang/ApiCall/Calls/Util.h

```diff
@@ -87,8 +87,8 @@ namespace api_call {
 clang::APValue const * evaluateNumber(
   const std::string debugTag, 
   const clang::Expr *expr,
-  const clang::ASTContext &Ctx
-) {
+  const clang::ASTContext &Ctx,
+  bool debugPrint=true) {
```

Comment:
nitpick: put `) {` on a separate line for consistency. (we should really figure out how to get clang-format running via GitHub Actions!)

Reply:
done